datumaro/cli/__main__.py

```diff
@@ -91,6 +89,30 @@ def make_parser():
         ('validate', commands.validate, "Validate project")
     ]
 
+def _get_params_with_paths():
```

Comment:
Probably, this function is better be renamed to a more general term. Something like "private args" or "sensitive args".